core/LSU.cpp

```diff
@@ -1,9 +1,14 @@
-
 #include "sparta/utils/SpartaAssert.hpp"
 #include "LSU.hpp"
 
 namespace olympia
 {
+    inline core_types::RegFile determineRegisterFile(const mavis::OperandInfo::Element & reg)
```

Comment:
Can you move this handy method to CoreUtils.hpp?

Reply:
Addressed in [4fd928d](https://github.com/riscv-software-src/riscv-perf-model/pull/51/commits/4fd928d4400af49de00ac3f5f8b5c4990bdbb2fd)